src/enclave/provider.ts

```diff
@@ -3,7 +3,7 @@
 
 import WebSocket from "isomorphic-ws";
 
-export const UninitializedConn = new Error("uninitialized connection");
+export const UninitializedConn = "uninitialized connection";
```

Comment:
The reason we had a constant `Error` was to allow easy comparisons on the user side.

`try { ... } catch (e) { if (e === sdk.UninitializedConn) /* do something nice */ }`

What was your reason behind the change?

Reply:
The error's stacktrace is lost because it's set in the constructor, not upon throwing, as far as I know. I'm not sure what the best course of action is, here. I'd recommend doing a check whether `e.message === UninitializedConn`.

Reply:
I will just create an issue in which we relay this discussion. I am okay with it for now, since we do not have any dependencies, which rely on the old behavior right now.

Tracked in #78